torch_tvm/operators.cpp

```diff
@@ -212,8 +279,8 @@ RegisterTVMOperator reg({
          auto bias_add_op = tvm::relay::Op::Get("nn.bias_add");
          auto bias_add_attrs = tvm::make_node<tvm::relay::BiasAddAttrs>();
          bias_add_attrs->axis = 1;
-         return tvm::relay::CallNode::make(
-             bias_add_op, {out, inputs[2]}, tvm::Attrs(bias_add_attrs), {});
+         return tvm::relay::Expr(tvm::relay::CallNode::make(
```

Comment:
we shouldn't need to wrap CallNode with Expr

Reply:
Thats what I thought, but it did not work in the first place. I changed the signature of the wrapper functions and now it seems fine.